app/scripts/modules/core/src/presentation/forms/inputs/TextInput.tsx

```diff
@@ -0,0 +1,31 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+import { IFormInputProps } from '../interface';
+
+interface ITextInputProps extends IFormInputProps, React.InputHTMLAttributes<any> {
+  inputClassName?: string;
+}
+
+export class TextInput extends React.Component<ITextInputProps> {
+  public render() {
+    const { field, validation, inputClassName, ...otherProps } = this.props;
+    const fieldProps = { ...field, value: field.value || '' };
+
+    const validationClassName = classNames({
+      'form-control': true,
+      'ng-dirty': validation.touched,
```

Comment:
are there plans to move away from using `ng-*` classes?

Reply:
Yes, I plan to merge @gcomstock designs + this PR +styleguide in the near future.